tests/examples/test_pass_thru.py

```diff
@@ -0,0 +1,55 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Move this file into a subdirectory under `tests/examples` to mirror the example it is testing (`developer_guide`)